include/swift/AST/DiagnosticsParse.def

```diff
@@ -762,6 +762,13 @@ ERROR(extra_rbracket,PointsToFirstBadToken,
       "unexpected ']' in type; did you mean to write an array type?", ())
 ERROR(extra_colon,PointsToFirstBadToken,
       "unexpected ':' in type; did you mean to write a dictionary type?", ())
+WARNING(subscript_array_element, none,
+        "unexpected subscript in array literal; did you mean to write two "
+        "separate elements instead?", ())
+NOTE(subscript_array_element_fix_it_add_comma, none, "add a separator between "
```

Comment:
Saying "a separator" is one level of indirection from what's actually meant, which can decrease comprehension particularly for non-native speakers; we can just state straight up that it's a comma:

"add ',' between elements" ("the" is unnecessary in [Swift diagnostics](https://github.com/apple/swift/blob/master/docs/Diagnostics.md), which uses a form of abbreviated English known as [reduced written register](http://folk.ntnu.no/andrewww/ICL-weir-pres-handout.pdf)).